Library/Formula/sdl_ttf.rb

```diff
@@ -18,6 +18,10 @@ class SdlTtf < Formula
   depends_on "sdl"
   depends_on "freetype"
 
+  # Fix broken TTF_RenderGlyph_Shaded()
+  # https://bugzilla.libsdl.org/show_bug.cgi?id=1433
```

Comment:
Yes, it's fixed in the latest version, 2.0.12, which is only compatible with SDL2. Homebrew has `sdl2_ttf` for that. The last version released for SDL1 is 2.0.11. The codebase has gone complete transition to SDL2 and it's unlikely to have another release for SDL1.